Transformers/Transformers.ipynb

```diff
@@ -32,7 +32,8 @@
    "metadata": {},
```

Comment:
<p>Let's run both LM finetuning and just an adapter both please? </p><p> </p><p>Finetuning the LM runs the risk of catastrophic forgetting and there is no way to tell if that has happened yet. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<blockquote>Let's run both LM finetuning and just an adapter both please?</blockquote><p>I don't understand. </p><p> </p>